spec/index.bs

```diff
@@ -2089,7 +2096,21 @@ enum GPUTextureAspect {
 
         **Returns:** |view|, of type {{GPUTextureView}}.
 
-        Issue: write definition. |this| |descriptor| |view|
+        1. If any of the following conditions are unsatisfied, generate a validation
+            error and return an [=invalid=] {{GPUTextureView}}.
+            <div class=validusage>
+                - 0 <= |baseMipLevel| < {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 <= |mipLevelCount| <= {{GPUTextureDescriptor/mipLevelCount}} - |baseMipLevel|
+                - 0 <= |baseArrayLayer| < {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 <= |arrayLayerCount| <= {{GPUTextureDescriptor/size.z}} - |baseArrayLayer|
+                - if {{GPUTextureDescriptor/dimension}} is "1d", |dimension| is "1d"
+                - if {{GPUTextureDescriptor/dimension}} is "2d", |dimension| is "2d"
+                - if {{GPUTextureDescriptor/dimension}} is "3d", |dimension| is in ["2d-array", "cube", "cube-array", "3d"]
+                - if |arrayLayerCount| > 1, |dimension| is in ["2d-array", "cube", "cube-array"]
+                - |format| must be compatible with GPUTexture's format.
```

Comment:
Right now it must be the same, we didn't add the mutable format flag to textures yet.